### DISCUSSION TITLE
Higher tier modules for space exploration maybe?

### DISCUSSION THREAD

**Discussion by TMPringles** [Xman1109/Atomic_Overhaul#21] [Ideas]
Maybe some new modules/higher tier thorium modules with high costs to balance out how insane a tier 9 version of the thorium module would be. c:

**Comment by Xman1109** [comment]
Not a Bad Idea, I will put it on the Todo list.

**Comment by NightOps007** [comment]
Also might want to make it so that lower tiered modules can be upgraded to the next higher tier module. Or make it so that you can disassemble the lower tiered ones to get the crafting ingredients back for use in making higher tiered modules. Maybe also add in other combined modules? i.e: speed/productivity(already have), speed/efficiency, productivity/efficiency, speed/productivity/efficiency. An idea to make such modules more precious would be to have them be crafted with depleted fuel cells of increasingly higher tiers depending on the tier of module instead, thus making it a must to have working reactor setups.
